envconfig.go

```diff
@@ -272,6 +276,12 @@ func generateFieldHandler(i int, tag envTag, typeHandler FieldTypeHandler) func(
 // ParseFromEnv populates structPtr from environment variables, returning warnings and fatal errors.  It panics if
 // structPtr is of the wrong type for this parser.
 func (p StructParser) ParseFromEnv(structPtr interface{}) (warn, fatal []error) {
+	return p.ParseUsingLookup(structPtr, os.LookupEnv)
+}
+
+// ParseUsingLookup populates structPtr from values returned by the given LookupEnv function, returning warnings and
+// fatal errors. It panics if structPtr is of the wrong type for this parser.
+func (p StructParser) ParseUsingLookup(structPtr interface{}, lookup LookupEnv) (warn, fatal []error) {
```

Comment:
TBH, I'd just have this replace `ParseFromEnv`. Especially since I'm worried that #1 is a breaking change for apro.git; emphasize that there are breaking changes going in.

Nit on naming: `s/Using/From/` it's parsing the results from `Lookup`; it's not using `Lookup` to parse something.